sigs.yml

```
@@ -0,0 +1,502 @@
+- name: Specification SIGs
+  sigs:
+  - name: 'Specification: General'
+    meeting: Every Tuesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo
+    chat:
+      - type: slack
+        name: '#otel-specification'
+        id: C01N7PP1THC
+    invites: calendar-spec-general
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Specification: Logs'
+    meeting: 'Merged into "Specification: General"'
+    notes:
+      type: none
+      value: See above
+    chat:
+      - type: slack
+        name: '#otel-logs'
+        id: C01N5UCHTEH
+    invites: none
+    tcSponsors:
+    - name: Tigran Najaryan
+      github: tigrannajaryan
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

@tigrannajaryan do you want to keep this as an independent SIG, or completely merge it into the specification SIG?

---

It should be merged. We already dissolved Log SIG and the description already says `Merged into "Specification: General"`. I am not sure if we want to keep the line separate or just delete it.

---

I will remove it then